packages/payload/src/fields/config/sanitize.ts

```diff
@@ -44,8 +53,41 @@ export const sanitizeFields = ({
     }
 
     // Make sure that the richText field has an editor
-    if (field.type === 'richText' && !field.editor && config.editor) {
-      field.editor = config.editor
+    if (field.type === 'richText') {
+      if (!field.editor && config.editor) {
+        field.editor = config.editor
+      }
+
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      const { i18nServerToUse, serverEditor } = field?.editor?.i18nServer
+        ? { i18nServerToUse: field?.editor?.i18nServer, serverEditor: field?.editor }
+        : { i18nServerToUse: config?.editor?.i18nServer, serverEditor: config?.editor }
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      const { clientEditor, i18nClientToUse } = field?.editor?.i18nClient
```

Comment:
Should this be skipped if initializing Payload with `localOnly`?

I'm unclear on the move to i18nServer vs i18nClient.